03-Custom-Actions-In-Python/03-Custom-Action-Features.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>I think this example is trying to demonstrate that if the attach method is overwritten the custom action breaks. But it would be confusing to a new user here--especially since the tutorial haven't shown a complex example of a custom action yet.</p><p>It may be useful for explaining how custom actions work under the hood, but I think a user might want to see more complex correct uses first.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>